src/libsync/owncloudpropagator.h

```diff
@@ -506,6 +506,10 @@ class OwncloudPropagator : public QObject
     bool createConflict(const SyncFileItemPtr &item,
         PropagatorCompositeJob *composite, QString *error);
 
+
+    QMap<QString, QString> _renamedDirectories;
```

Comment:
Will not splitting this into local and remote renames lead to bugs similar to in the discovery? 

Please add a explanatory comment too: Why is this necessary when `adjustRenamedPath()` is already called on _item->_file during discovery?

Reply:
> Will not splitting this into local and remote renames lead to bugs similar to in the discovery?

No, because in the discovery we map  original -> current local or server path, which might be different from target.

But here we always map origin->target and that is the same for both local and server.

I'm adding a comment